resolver-fundamentals.mdx

```diff
@@ -1,52 +0,0 @@
----
```

Comment:
we should be wary of deleting any docs that we may have linked to in chalkpy or CLI